From 38f636273b1a20bf4db7bcadd2c2b948271ba4bb Mon Sep 17 00:00:00 2001
From: Antonio Borneo <antonio.borneo@foss.st.com>
Date: Fri, 13 Jan 2023 10:07:20 +0100
Subject: [PATCH 0335/1695] plat-stm32mp2: conf: enable EXTI for stm32mp2
 platforms

Default enable stm32_exti driver for stm32mp2 platforms.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git f82e7f2ae2cc7820a6872423a74bbcabf31b49cb plat-stm32mp2: conf: enable EXTI for stm32mp2 platforms]
Signed-off-by: Antonio Borneo <antonio.borneo@foss.st.com>
Change-Id: If5f8f07e9e4e8c243978784537d143345c27c4de
---
 core/arch/arm/plat-stm32mp2/conf.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/core/arch/arm/plat-stm32mp2/conf.mk b/core/arch/arm/plat-stm32mp2/conf.mk
index 7dc51deb5..c780cc67f 100644
--- a/core/arch/arm/plat-stm32mp2/conf.mk
+++ b/core/arch/arm/plat-stm32mp2/conf.mk
@@ -67,6 +67,7 @@ CFG_TEE_CORE_NB_CORE ?= 2
 CFG_STM32MP_OPP_COUNT ?= 3
 
 CFG_STM32_BSEC3 ?= y
+CFG_STM32_EXTI ?= y
 CFG_STM32_GPIO ?= y
 CFG_STM32_IAC ?= y
 CFG_STM32_RIF ?= y
-- 
2.47.2

